bgpd/bgp_damp.h

```diff
@@ -145,9 +145,7 @@ extern int bgp_damp_withdraw(struct bgp_path_info *path, struct bgp_dest *dest,
 			     afi_t afi, safi_t safi, int attr_change);
 extern int bgp_damp_update(struct bgp_path_info *path, struct bgp_dest *dest,
 			   afi_t afi, safi_t saff);
-extern void bgp_damp_info_free(struct bgp_damp_info *bdi,
-			       struct bgp_damp_config *bdc, int withdraw,
-			       afi_t afi, safi_t safi);
+extern void bgp_damp_info_free(struct bgp_damp_info *bdi, int withdraw);
```

Comment:
Since we are here, can we convert to bool (withdraw)?

Reply:
There is more to clean up in this code, and I am ready to work on it next week.
But would be great to merge this one if you're fine with it because it needs to be backported.

Reply:
Sure.